packages/admin/cms-admin/src/blocks/rte/extension/CmsLink/createCmsLinkToolbarButton.tsx

```diff
@@ -131,22 +131,48 @@ export function createCmsLinkToolbarButton({ link: LinkBlock }: CreateCmsLinkToo
                 <DialogContent>
                     <LinkBlock.AdminComponent state={newLinkState} updateState={setNewLinkState} />
                 </DialogContent>
-                <DialogActions>
-                    {linkEntity && (
-                        <Button onClick={handleRemove} color="primary">
-                            <FormattedMessage id="comet.rteExtensions.cmsLink.removeLink" defaultMessage="Delete Link" />
-                        </Button>
-                    )}
-                    <Button onClick={handleUpdate} color="primary">
-                        <FormattedMessage {...messages.ok} />
-                    </Button>
-                    <Button onClick={handleClose} color="primary">
+                <StyledDialogActions>
+                    <CancelButton onClick={handleClose} color="primary">
                         <FormattedMessage {...messages.cancel} />
-                    </Button>
-                </DialogActions>
+                    </CancelButton>
+                    <ButtonContainer>
+                        {linkEntity && (
+                            <DeleteButton onClick={handleRemove} color="primary">
+                                <FormattedMessage id="comet.rteExtensions.cmsLink.removeLink" defaultMessage="Delete Link" />
+                            </DeleteButton>
+                        )}
+                        <OkayButton onClick={handleUpdate} color="primary">
+                            <FormattedMessage {...messages.ok} />
+                        </OkayButton>
+                    </ButtonContainer>
```

Comment:
You shoudn't need to set the `color` for any of the predefined Button components. 
Also the custom Text-Content should only be needed for "Delete Link". "Cancel" and "OK" are already set the the predefined components.